app/objects/score.py

```diff
@@ -180,20 +182,25 @@ def __init__(self):
         self.perfect: bool
         self.status: SubmissionStatus
 
-        self.play_time: datetime
+        self.client_time: datetime
+        self.server_time: datetime
         self.time_elapsed: int
 
         self.client_flags: ClientFlags
-        self.online_checksum: str
+        self.client_checksum: str
 
         self.rank: Optional[int] = None
         self.prev_best: Optional[Score] = None
 
-    def __repr__(self) -> str:  # maybe shouldn't be so long?
-        return (
-            f"<{self.acc:.2f}% {self.max_combo}x {self.nmiss}M "
-            f"#{self.rank} on {self.bmap.full_name} for {self.pp:,.2f}pp>"
-        )
+    def __repr__(self) -> str:
+        # TODO: i really need to clean up my reprs
+        try:
+            return (
+                f"<{self.acc:.2f}% {self.max_combo}x {self.nmiss}M "
+                f"#{self.rank} on {self.bmap.full_name} for {self.pp:,.2f}pp>"
+            )
+        except:
+            return super().__repr__()
```

Comment:
this is technically only required since i'm doing generate_appropriate_stacktrace before these params are populated